Traceback (most recent call last):
  File "/root/autodl-tmp/index-tts-vllm/server/task_worker.py", line 24, in <module>
    from indextts.infer_vllm import IndexTTS
  File "/root/autodl-tmp/index-tts-vllm/indextts/infer_vllm.py", line 22, in <module>
    from indextts.gpt.model_vllm import UnifiedVoice
  File "/root/autodl-tmp/index-tts-vllm/indextts/gpt/model_vllm.py", line 4, in <module>
    import patch_vllm  # ⚠️ Monkey Patch, do not delete this line
    ^^^^^^^^^^^^^^^^^
  File "/root/autodl-tmp/index-tts-vllm/patch_vllm.py", line 6, in <module>
    vllm_version = version.parse(importlib.import_module("vllm").__version__)
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'vllm'
Traceback (most recent call last):
  File "/root/autodl-tmp/index-tts-vllm/server/task_worker.py", line 24, in <module>
    from indextts.infer_vllm import IndexTTS
  File "/root/autodl-tmp/index-tts-vllm/indextts/infer_vllm.py", line 22, in <module>
    from indextts.gpt.model_vllm import UnifiedVoice
  File "/root/autodl-tmp/index-tts-vllm/indextts/gpt/model_vllm.py", line 4, in <module>
    import patch_vllm  # ⚠️ Monkey Patch, do not delete this line
    ^^^^^^^^^^^^^^^^^
  File "/root/autodl-tmp/index-tts-vllm/patch_vllm.py", line 6, in <module>
    vllm_version = version.parse(importlib.import_module("vllm").__version__)
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'vllm'
Traceback (most recent call last):
  File "/root/autodl-tmp/index-tts-vllm/server/task_worker.py", line 24, in <module>
    from indextts.infer_vllm import IndexTTS
  File "/root/autodl-tmp/index-tts-vllm/indextts/infer_vllm.py", line 22, in <module>
    from indextts.gpt.model_vllm import UnifiedVoice
  File "/root/autodl-tmp/index-tts-vllm/indextts/gpt/model_vllm.py", line 4, in <module>
    import patch_vllm  # ⚠️ Monkey Patch, do not delete this line
    ^^^^^^^^^^^^^^^^^
  File "/root/autodl-tmp/index-tts-vllm/patch_vllm.py", line 6, in <module>
    vllm_version = version.parse(importlib.import_module("vllm").__version__)
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'vllm'
Traceback (most recent call last):
  File "/root/autodl-tmp/index-tts-vllm/server/task_worker.py", line 24, in <module>
    from indextts.infer_vllm import IndexTTS
  File "/root/autodl-tmp/index-tts-vllm/indextts/infer_vllm.py", line 22, in <module>
    from indextts.gpt.model_vllm import UnifiedVoice
  File "/root/autodl-tmp/index-tts-vllm/indextts/gpt/model_vllm.py", line 4, in <module>
    import patch_vllm  # ⚠️ Monkey Patch, do not delete this line
    ^^^^^^^^^^^^^^^^^
  File "/root/autodl-tmp/index-tts-vllm/patch_vllm.py", line 6, in <module>
    vllm_version = version.parse(importlib.import_module("vllm").__version__)
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'vllm'
INFO 09-26 15:48:51 [__init__.py:243] Automatically detected platform cuda.
✅  Registry GPT2TTSModel to vllm
✅  SamplingParams._verify_args Patched
✅  ModelInputForGPUBuilder._compute_lens Patched
Traceback (most recent call last):
  File "/root/autodl-tmp/index-tts-vllm/server/task_worker.py", line 29, in <module>
    from .database.db_manager import DatabaseManager
ImportError: attempted relative import with no known parent package
INFO 09-26 15:49:01 [__init__.py:243] Automatically detected platform cuda.
✅  Registry GPT2TTSModel to vllm
✅  SamplingParams._verify_args Patched
✅  ModelInputForGPUBuilder._compute_lens Patched
Traceback (most recent call last):
  File "/root/autodl-tmp/index-tts-vllm/server/task_worker.py", line 29, in <module>
    from .database.db_manager import DatabaseManager
ImportError: attempted relative import with no known parent package
INFO 09-26 15:49:13 [__init__.py:243] Automatically detected platform cuda.
✅  Registry GPT2TTSModel to vllm
✅  SamplingParams._verify_args Patched
✅  ModelInputForGPUBuilder._compute_lens Patched
Traceback (most recent call last):
  File "/root/autodl-tmp/index-tts-vllm/server/task_worker.py", line 29, in <module>
    from .database.db_manager import DatabaseManager
ImportError: attempted relative import with no known parent package
INFO 09-26 15:49:27 [__init__.py:243] Automatically detected platform cuda.
✅  Registry GPT2TTSModel to vllm
✅  SamplingParams._verify_args Patched
✅  ModelInputForGPUBuilder._compute_lens Patched
Traceback (most recent call last):
  File "/root/autodl-tmp/index-tts-vllm/server/task_worker.py", line 29, in <module>
    from .database.db_manager import DatabaseManager
ImportError: attempted relative import with no known parent package
INFO 09-26 15:49:37 [__init__.py:243] Automatically detected platform cuda.
✅  Registry GPT2TTSModel to vllm
✅  SamplingParams._verify_args Patched
✅  ModelInputForGPUBuilder._compute_lens Patched
Traceback (most recent call last):
  File "/root/autodl-tmp/index-tts-vllm/server/task_worker.py", line 29, in <module>
    from .database.db_manager import DatabaseManager
ImportError: attempted relative import with no known parent package
INFO 09-26 15:49:49 [__init__.py:243] Automatically detected platform cuda.
✅  Registry GPT2TTSModel to vllm
✅  SamplingParams._verify_args Patched
✅  ModelInputForGPUBuilder._compute_lens Patched
Traceback (most recent call last):
  File "/root/autodl-tmp/index-tts-vllm/server/task_worker.py", line 29, in <module>
    from .database.db_manager import DatabaseManager
ImportError: attempted relative import with no known parent package
INFO 09-26 15:50:01 [__init__.py:243] Automatically detected platform cuda.
✅  Registry GPT2TTSModel to vllm
✅  SamplingParams._verify_args Patched
✅  ModelInputForGPUBuilder._compute_lens Patched
Traceback (most recent call last):
  File "/root/autodl-tmp/index-tts-vllm/server/task_worker.py", line 29, in <module>
    from .database.db_manager import DatabaseManager
ImportError: attempted relative import with no known parent package
INFO 09-26 15:50:12 [__init__.py:243] Automatically detected platform cuda.
✅  Registry GPT2TTSModel to vllm
✅  SamplingParams._verify_args Patched
✅  ModelInputForGPUBuilder._compute_lens Patched
Traceback (most recent call last):
  File "/root/autodl-tmp/index-tts-vllm/server/task_worker.py", line 29, in <module>
    from .database.db_manager import DatabaseManager
ImportError: attempted relative import with no known parent package
INFO 09-26 15:50:23 [__init__.py:243] Automatically detected platform cuda.
✅  Registry GPT2TTSModel to vllm
✅  SamplingParams._verify_args Patched
✅  ModelInputForGPUBuilder._compute_lens Patched
Traceback (most recent call last):
  File "/root/autodl-tmp/index-tts-vllm/server/task_worker.py", line 29, in <module>
    from .database.db_manager import DatabaseManager
ImportError: attempted relative import with no known parent package
INFO 09-26 15:50:34 [__init__.py:243] Automatically detected platform cuda.
✅  Registry GPT2TTSModel to vllm
✅  SamplingParams._verify_args Patched
✅  ModelInputForGPUBuilder._compute_lens Patched
Traceback (most recent call last):
  File "/root/autodl-tmp/index-tts-vllm/server/task_worker.py", line 29, in <module>
    from .database.db_manager import DatabaseManager
ImportError: attempted relative import with no known parent package
INFO 09-26 15:50:47 [__init__.py:243] Automatically detected platform cuda.
✅  Registry GPT2TTSModel to vllm
✅  SamplingParams._verify_args Patched
✅  ModelInputForGPUBuilder._compute_lens Patched
Traceback (most recent call last):
  File "/root/autodl-tmp/index-tts-vllm/server/task_worker.py", line 29, in <module>
    from .database.db_manager import DatabaseManager
ImportError: attempted relative import with no known parent package
